src/DssSpell.t.sol

```diff
@@ -49,16 +49,67 @@ interface SequencerLike {
     function hasJob(address job) external view returns (bool);
 }
 
-interface RwaLiquidationOracleLike {
-    function good(bytes32 ilk) external view returns (bool);
-    function ilks(bytes32) external view returns (string memory doc, address pip, uint48 tau, uint48 toc);
+interface L1TokenBridgeLike {
+    function l1ToL2Token(address) external view returns (address);
+    function isOpen() external view returns (uint256);
+    function escrow() external view returns (address);
+    function otherBridge() external view returns (address);
+    function messenger() external view returns (address);
+    function version() external view returns (string memory);
+    function getImplementation() external view returns (address);
+    function bridgeERC20To(
+        address _localToken,
+        address _remoteToken,
+        address _to,
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes memory _extraData
+    ) external;
+}
+
+interface L2TokenBridgeLike {
+    function l1ToL2Token(address) external view returns (address);
+    function isOpen() external view returns (uint256);
+    function escrow() external view returns (address);
+    function otherBridge() external view returns (address);
+    function messenger() external view returns (address);
+    function version() external view returns (string memory);
+    function maxWithdraws(address) external view returns (uint256);
+    function getImplementation() external view returns (address);
+    function bridgeERC20To(
+        address _localToken,
+        address _remoteToken,
+        address _to,
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes memory _extraData
+    ) external;
+}
+
+interface BaseGovRelayLike {
```

Comment:
I would split this into the L1 and L2 interfaces as they are slightly different.